.github/workflows/kernel-demos.yml

```diff
@@ -9,8 +9,8 @@ jobs:
       - name: Checkout the FreeRTOS/FreeRTOS Repository
         uses: actions/checkout@v3
         with:
-          ref: main
-          repository: FreeRTOS/FreeRTOS
+          ref: msvcDemoWinsockTwo
```

Comment:
This is not recommended.

Reply:
Currently the changes here break the existing MSVC demos in [FreeRTOS/FreeRTOS/Demo/WIN32-MSVC](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS/Demo/WIN32-MSVC), and [FreeRTOS/FreeRTOS/Demo/WIN32-MSVC-Static-Allocation-Only](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS/Demo/WIN32-MSVC-Static-Allocation-Only).
We'll need to merge a "failing" PR into here, or into FreeRTOS/FreeRTOS. For now I have this pointed to my fork to show the changes to this port work with the changes to the demo projects.